vendor/constants.js

```diff
@@ -44,7 +44,7 @@ module.exports = function(babel) {
           // into this:
           //
           // if (!condition) {
-          //   if ("production" !== process.env.NODE_ENV) {
+          //   if ('production' !== process.env.NODE_ENV) {
```

Comment:
Can you revert this file (it actually does end up as double quotes because babel)